cfme/infrastructure/host.py

```diff
@@ -329,6 +303,66 @@ def run_smartstate_analysis(self):
         sel.handle_alert()
         flash.assert_message_contain('"{}": Analysis successfully initiated'.format(self.name))
 
+    def check_compliance(self):
+        """ Checks compliace on this host
+
+        Note:
+            The host must have assigned policy profile with host compliance policy.
+        """
+        navigate_to(self, 'Details')
+        tb.select('Policy', 'Check Compliance of Last Known Configuration', invokes_alert=True)
+        sel.handle_alert()
+        flash.assert_no_errors()
+
+    @contextmanager
```

Comment:
This is interesting idea and I kinda like it, though I think it is a methodology that we should explore fully before we start using it. Thoughts ?  @mfalesni @RonnyPfannschmidt @jkrocil @izapolsk @mshriver @otsuman